onboarding/chat-moderators.md

```diff
@@ -13,10 +13,21 @@ Removal of the role is subject to the discretion of the Core team members and ma
 
 ## Chat Responsibilities and Guidelines
 
+### Take notes of any moderation action
+- The Discord server uses [Dyno bot](https://dyno.gg/bot) to aid in automation of certain moderation tasks,
+- When you moderate a user, you should add a note on the user in the `#chat-mod-actions` channel explaining the rule they broke and what action was taken (e.g. warned, threaded, etc).
+- You can include links to messages in another channel within user notes, if you feel the context of the message is relevant to the note.
+- You can check out some of the available [Dyno commands](https://dyno.gg/commands), but the ones you will most likely be using most often are `?notes` and `?note`.
+
 ### Maintain, follow, and enforce rules
 
 - As a moderator, you will be expected to **adhere to** and **uphold** the rules as detailed and maintained in the `#rules` channel in the Discord at all times. 
-- **`Delete` messages from all channels and threads.**
+- **Change inappropriate nicknames**
+    - User nicknames _should_ adhere to our rules.
+    - Overtly hateful, racist, or sexual names are not tolerated.
+    - If possible, change the user's name to something more appropriate and let the user know of the name change in a private thread
+    - Abuse of this permission will not be tolerated.
+- **Delete applicable messages from all channels and threads.**
```

Comment:
I believe that adding it as another small section under inappropriate nicknames would be a good placement. The changing inappropriate nicknames section mentions that overtly hateful, racist, or sexual names are not tolerated which would make sense before this new section. I think an example of the wording for emphasizing intention vs impact could be.
"-  **Intent vs. Impact for disparaging comments**
      -- Users _should_ be reminded that we are an inclusive community and agree in cases where comments are toxic towards a user, individual, or group characteristics.
      -- Given a toxic name or toxic comments they should be reminded people do read comments.
      -- It's a good idea to remind User's even if their _intent isn't malicious that_ other users reading may be impacted negatively.
      -- An example is racist or hateful names going against our goal of having an inclusive comment and can impact other users in a negative way making it seem they aren't welcome.

Reply:
Updated the PR. Let me know if what I committed in is along the lines of what you were thinking or if you think it could be tweaked further to solidify that message.